sql/sql_lex.h

```diff
@@ -655,10 +655,10 @@ class st_select_lex_unit: public Sql_alloc
   bool prepare(THD *thd, Query_result *result, ulonglong added_options,
                ulonglong removed_options);
   bool optimize(THD *thd);
-  //TIANMU UPGRADE END
+  
   int optimize_for_tianmu();
-  int optimize_after_tianmu();
-  //END
+  int after_optimize_tianmu();
```

Comment:
why rename the function name? what's the difference?

Reply:
The `optimize_after_tianmu`  has been as a variable name，so I think it is not suitable as a function name.

Reply:
a function name should start with a verb. like set_value, update_row, I suggest rename the variable name.